sdks/rust/src/protocol/payload.rs

```diff
@@ -2342,7 +2342,11 @@ impl SabrePayloadBuilder {
 
         Ok(TransactionBuilder::new()
             .with_family_name("sabre".into())
-            .with_family_version(env!("CARGO_PKG_VERSION").into())
+            .with_family_version(format!(
+                "{}.{}",
+                env!("CARGO_PKG_VERSION_MAJOR"),
+                env!("CARGO_PKG_VERSION_MINOR")
+            ))
```

Comment:
family version is the protocol the smart contract adheres to and should only really change when it is incompatible with previous versions.

Reply:
Isn't that implied by a major/minor version change? Not sure what change you are requesting.

Reply:
The family version isn't the same as the package version (of sabre), it's more like an API version of the smart contract itself.